application/client/src/app/ui/views/toolbar/chart/zoomer/cursor/state.ts

```diff
@@ -0,0 +1,274 @@
+import { InternalAPI } from '@service/ilc';
+import { ViewContainerRef } from '@angular/core';
+import { Chart, IPositionChange } from '@ui/service/toolbar/chart';
+
+enum EChangeKind {
+    move = 'move',
+    left = 'left',
+    right = 'right',
+    undefined = 'undefined',
+}
+
+const CSettings = {
+    minSize: 20,
+};
```

Comment:
for single constant doesn't make sense to create object / group... better would be 
```
const MIN_SIZE_OF_SOMETHING = 20;
```